tests/_test_utils/test_mssautoplot.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""
+
+    tests._test_utils.test_mssautoplot
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This module provides pytest functions to tests msui.msui
+
+    This file is part of MSS.
+
+    :copyright: Copyright 2017 Joern Ungermann
```

Comment:
there should be your name added and this year

Reply:
Done!